src/messaging/tests/TestExchangeMgr.cpp

```diff
@@ -189,7 +190,11 @@ nlTestSuite sSuite =
  */
 int Initialize(void * aContext)
 {
-    CHIP_ERROR err = gTransportMgr.Init("LOOPBACK");
+    CHIP_ERROR err = chip::Platform::MemoryInit();
```

Comment:
That's fine; maybe add a comment here about why we are not doing a shutdown call?

Reply:
This may just be a couple of simple bugs in existing Shutdowns rather than a pervasive omission of deallocations. I'll do a bit more investigation before deciding whether to defer this to a followup.

Reply:
Fixed.